docs/content/en/docs/getting-started/site-engineer/_index.md

```diff
@@ -0,0 +1,232 @@
+    kube-prometheus-stack:
+      prometheus:
+        enabled: true
+        prometheusSpec:
+          externalUrl: http://localhost:9090/prometheus
+          routePrefix: /prometheus
+        ingress:
+          enabled: true
+          paths:
+            - /prometheus
+      grafana:
+        enabled: true
+        ingress:
+          enabled: true
+          path: /grafana
+        adminUser: admin
+        adminPassword: admin123
+        grafana.ini:
+          server:
+            root_url: http://localhost:3000/grafana
+           serve_from_sub_path: true
+    ```
+
+1. Copy the following YAML into a file named `traefik.values.yaml`:
+
+    ```yaml
+    ---
+    providers:
+      kubernetesCRD:
+        namespaces:
+          - default
+          - traefik
+      kubernetesIngress:
+        namespaces:
+          - default
+          - traefik
+
+    ports:
+      traefik:
+        expose: true
+        nodePort: 32090
+      web:
+        nodePort: 32080
+      websecure:
+        nodePort: 32443
+      cassandra:
+        expose: true
+        port: 9042
+        nodePort: 32091
+      cassandrasecure:
+        expose: true
+        port: 9142
+        nodePort: 32092
+      sg-graphql:
+        expose: true
+        port: 8080
+        nodePort: 30080
+      sg-auth:
+        expose: true
+        port: 8081
+        nodePort: 30081
+      sg-rest:
+        expose: true
+        port: 8082
+        nodePort: 30082
+
+    service:
+      type: NodePort
+    ```
+
+1. Install Traefik:
+
+    ```bash
+    helm install -f traefik.values.yaml traefik traefik/traefik -n traefik --create-namespace
+    NAME: traefik
+    LAST DEPLOYED: Fri Feb 19 12:40:36 2021
+    NAMESPACE: traefik
+    STATUS: deployed
+    REVISION: 1
+    TEST SUITE: None
+    ```
+
+1. Upgrade the K8ssandra pod with Traefik support:
+
+    ```bash
+    helm upgrade -f k8ssandra-traefik.values.yaml k8ssandra k8ssandra/k8ssandra
+    Release "k8ssandra" has been upgraded. Happy Helming!
+    NAME: k8ssandra
+    LAST DEPLOYED: Fri Feb 19 12:46:49 2021
+    NAMESPACE: default
+    STATUS: deployed
+    REVISION: 2
+    ```
+
+1. Verify that the Traefik pod is `Running`:
+
+    ```bash
+    kubectl get po -A | grep "traefik"
+    traefik       traefik-55996cbb6-v6s9p                1/1     Running     0          13m
+    ```
+
+Now you're ready to access the full suite of K8ssandra supporting applications.
+
+{{% alert title="Tip" color="success" %}}
+If you've followed the configuration instructions in this quick start, the URLs in the sections below should take you directly to the associated utilities.
+{{% /alert %}}
+
+## Access K8ssandra utilities
+
+K8ssandra includes the following bundled and customized utilities:
+
+### Cassandra Reaper
+
+[Cassandra Reaper](<"http://repair.127.0.0.1.nip.io:8080/webui">) provides an easy interface for managing K8ssandra cluster repairs. For details, see the [Cassandra Reaper](http://cassandra-reaper.io/) web site.
```

Comment:
I was not expecting the first link to go to the reaper dashboard. I was actually expecting it to go to the reaper site. For comparison, the "Stargate" link in a similar section on the developer quick start goes to stargate.io.

Suggest actually providing the URL for the dashboard in the page text, as is done in the developer quick start for the Stargate healthcheck endpoint.

Along with the above, I'd make the second Cassandra Reaper link go to the task page.